egs2/jtubespeech/tts1/run.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+fs=16000
+n_fft=1024
+n_shift=256
+win_length=1024
+
+opts=
+if [ "${fs}" -eq 16000 ]; then
+    # To suppress recreation, specify wav format
+    opts="--audio_format wav "
+else
+    opts="--audio_format flac "
+fi
+
+train_set=tr_no_dev
+valid_set=dev
+test_sets=test
+
+train_config=conf/train.yaml
+inference_config=conf/decode.yaml
+
+# Input example: こ、こんにちは
+
+# 1. Phoneme + Pause
+# (e.g. k o pau k o N n i ch i w a)
+g2p=pyopenjtalk
+
+# 2. Kana + Symbol
+# (e.g. コ 、 コ ン ニ チ ワ)
+# g2p=pyopenjtalk_kana
+
+# 3. Phoneme + Accent
+# (e.g. k 1 0 o 1 0 k 5 -4 o 5 -4 N 5 -3 n 5 -2 i 5 -2 ch 5 -1 i 5 -1 w 5 0 a 5 0)
+# g2p=pyopenjtalk_accent
+
+# 4. Phoneme + Accent + Pause
+# (e.g. k 1 0 o 1 0 pau k 5 -4 o 5 -4 N 5 -3 n 5 -2 i 5 -2 ch 5 -1 i 5 -1 w 5 0 a 5 0)
+# g2p=pyopenjtalk_accent_with_pause
+
+./tts.sh \
```

Comment:
if you changed the default config to the config using x-vector, please add the option `--use_xvector true` for `tts.sh`.

Reply:
e.g.,
https://github.com/espnet/espnet/blob/3d83127d074c4af26cc86f30f0773f72a9ad5bbe/egs2/libritts/tts1/run.sh#L41

Reply:
Thanks for your detailed instruction. I edited conf/ and run.sh for the multi-speaker setting.